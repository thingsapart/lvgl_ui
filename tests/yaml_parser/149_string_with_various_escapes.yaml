# Tests more complex JSON-style escapes inside a double-quoted string.
message: "First line.\n\tSecond line, indented.\n\"A quote\" and a \\ backslash."
