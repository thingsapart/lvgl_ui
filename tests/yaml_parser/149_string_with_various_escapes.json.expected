[{
		"message":	"First line.\n\tSecond line, indented.\n\"A quote\" and a \\ backslash."
	}]
